CYD-Klipper/platformio.ini

```diff
@@ -84,3 +84,13 @@ board = esp32-4827S043C-smartdisplay
 [env:esp32-8048S043C-SD]
 board = esp32-8048S043C-smartdisplay
 
+[env:esp32-CROWPANEL-28R]
+platform = https://github.com/platformio/platform-espressif32#v6.9.0
```

Comment:
Why is a newer platform needed?

Reply:
v6.4.0 is not working throws error while compiling, so moved to the lasted release

Reply:
Which errors?

Reply:
*  Executing task: C:\Users\Mad-Rajib\.platformio\penv\Scripts\platformio.exe run --environment esp32-2432S028R 

Processing esp32-2432S028R (board: esp32-2432S028R; platform: https://github.com/platformio/platform-espressif32#v6.4.0; framework: arduino)
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
**UnknownPlatform: Unknown development platform 'PackageItem <path=C:\Users\Mad-Rajib\.platformio\platforms\espressif32@src-9b86d13ed00bff4c3ffbb3904373b959 metadata=PackageMetadata <type=platform name=espressif32 version=6.4.0+sha.516520f spec={'owner': None, 'id': None, 'name': 'platform-espressif32', 'requirements': None, 'uri': 'git+https://github.com/platformio/platform-espressif32#v6.4.0'}'**

 *  The terminal process "C:\Users\Mad-Rajib\.platformio\penv\Scripts\platformio.exe 'run', '--environment', 'esp32-2432S028R'" terminated with exit code: 1. 
 *  Terminal will be reused by tasks, press any key to close it.

Reply:
Error was only on windows platform for some reason, in linux seems everything is working fine.